README.md

```diff
@@ -78,14 +94,13 @@ docker run \
   -v ./results:/usr/src/app/results \
   -e DATABASE_URL \
   ghcr.io/peopleforbikes/brokenspoke-analyzer:2.4.0 \
-  -vv export local-calver "united states" "santa rosa" "new mexico"
+  -vv export local "united states" "santa rosa" "new mexico"
 ```
 
 Clean up:
```

Comment:
A little bit of emphasis so that users know that this step is important before attempting another run.
```suggestion
Clean up (required before attempting to run another analysis):
```